projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss

```diff
@@ -90,14 +90,62 @@
     @include igx-tree-typography($type-scale);
 
     // Add theme type-scale specific quirks
-    @if ($_variant == 'material' or $_variant == 'fluent') {
+    @if ($_variant == 'material') {
         @include igx-calendar-typography($type-scale);
         @include igx-card-typography($type-scale);
         @include igx-dialog-typography($type-scale);
         @include igx-drop-down-typography($type-scale);
         @include igx-list-typography($type-scale);
     }
 
+    @if ($_variant == 'material' or $_variant == 'fluent') {
+        @include igx-calendar-typography($type-scale);
+        @include igx-dialog-typography($type-scale);
+        @include igx-drop-down-typography($type-scale);
+    }
+
+    @if ($_variant == 'fluent') {
+        @include igx-checkbox-typography($type-scale, $categories: (
+            label: 'body-2'
+        ));
+
+        @include igx-radio-typography($type-scale, $categories: (
+            label: 'body-2'
+        ));
+
+        @include igx-switch-typography($type-scale, $categories: (
+            label: 'body-2'
+        ));
+
+        @include igx-input-group-typography($type-scale, $categories: (
+            input-text: 'body-2'
+        ));
```

Comment:
just as a comment since I am not sure if you have noticed the label, it is outside the artboard and uses subtitle 2 scale for the label that floats above the input box

Reply:
subtitle-2 uses font-weight: 700, but the Microsoft fluent label is 600.

Reply:
yes, I am aware of that but I find them to be close enough, so that we don't have to add a specific handling for such a small difference